packages/issuer-sdk-js/src/create-issuer-config.test.ts

```diff
@@ -40,10 +42,12 @@ describe("createIssuerConfig", () => {
     const mockWallet = new Wallet(
       "dcdda6663be8dfa23d0e54a31ff6fddba2fdf8a1f0eae985c59857031e6da169",
     );
+    const mockSecretKey = nacl.randomBytes(32);
```

Comment:
Let's use this interface to make sure we get the right bytes:
```suggestion
    const mockSecretKey = nacl.box.keypair().secretKey;
```